spearhead/variables.css

```diff
@@ -52,6 +52,8 @@
 	--button--font-weight: bold;
 	--button--font-family: var(--global--font-secondary);
 	--button--font-size: var(--global--font-size-sm);
+	--button--padding-vertical: 24px;
+	--button--padding-horizontal: 28px;
```

Comment:
These look like variables that will only work well with that size for the button on the header menu, not the rest of the site's buttons (like the search one)

Reply:
Open to suggestions on better values here. But I used the search block's button and main button block as references to determine them. Are there instances where these are breaking button appearances?

<img width="782" alt="Screen Shot 2020-10-13 at 2 56 28 PM" src="https://user-images.githubusercontent.com/5375500/95903744-92758c80-0d64-11eb-96c0-78f173ac6619.png">

Reply:
Yeah, there’s a work in progress PR for the search block, @kjellr designed some of the possibilities, maybe we can use them for reference? https://github.com/Automattic/themes/pull/2564#issuecomment-707878367

Reply:
I should have opened with that, sorry!

Reply:
No worries, thanks! I took a look at those mockups and adjusted the variables to get close. We can iterate as needed in a follow up.